.github/workflows/deployment-pr-cleanup.yml

```diff
@@ -90,7 +90,7 @@ jobs:
           echo number=${BASH_REMATCH[1]} >> $GITHUB_OUTPUT
 
       - name: Comment about uninstallation on PR
-        uses: thollander/actions-comment-pull-request@v2
+        uses: thollander/actions-comment-pull-request@v3
         if: steps.pr-number.outputs.number
         with:
           pr_number: ${{ steps.pr-number.outputs.number }}
```

Comment:
according to https://github.com/thollander/actions-comment-pull-request/releases/tag/v3.0.0
we should rename these parameters:

- From pr_number to pr-number
- From comment_tag to comment-tag
- From create_if_not_exists to create-if-not-exists